packages/expo-ui/components/DatePicker/index.tsx

```diff
@@ -0,0 +1,98 @@
+import { requireNativeView } from 'expo';
+import { Platform } from 'expo-modules-core';
+import React from 'react';
+import { StyleProp, ViewStyle } from 'react-native';
+
+import { ViewEvent } from '../../src';
+
+type AndroidVariant = 'picker' | 'input';
+
+type IOSVariant = 'wheel' | 'automatic' | 'graphical' | 'compact';
+
+type DisplayedComponents = 'date' | 'hourAndMinute' | 'dateAndTime';
+
+/**
+ * Props for the DatePicker component.
+ */
+export type DatePickerProps = {
+  /**
+   * The intial date to display on the picker.
+   */
+  initialDate?: string | null;
+  /**
+   * A title displayed on the picker  on iOS.
```

Comment:
```suggestion
   * A title displayed on the picker on iOS.
```
nit, redundant space